integration_tests/steps/integration_tests.py

```diff
@@ -792,3 +802,22 @@ def test_config_parameters(context):
     assert grid.config.iaa_fee_const == 1
     assert all([rate == 35
                 for rate in grid.config.market_maker_rate.values()])
+
+
+@then('trades on the {market_name} market clear with {clearing_rate} cents/kWh')
+def trades_clear_price(context, market_name, clearing_rate):
```

Comment:
Here again, `clear_price` or `clearing_rate` does not match with the test for the one-sided and pab market structure. How about using  `trade_rate` ? Again, minor! Thanks

Reply:
Agreed, fixing it.